content/en/docs/profiles.md

```diff
@@ -83,6 +85,7 @@ We recommend this profile for those who fully trust their automation to renew th
 | [Validity Period](#validity-period)                                  | 160 hours |
 | [Revocation Information](#revocation-information)                    | CRL       |
 | [Max Names](#max-names)                                              | 25        |
+| [Identifier Types](#identifier-types)                                | DNS       |
```

Comment:
Actually, imo this should include IP already, since that's true in Staging and the public isn't on the Prod allowlist yet anyway.
```suggestion
| [Identifier Types](#identifier-types)                                | DNS, IP   |
```